src/scss/inspector.scss

```diff
@@ -1,42 +1,48 @@
-.property {
-  $can-drop-size: 14px;
-  padding:10px;
-  display: flex;
-  align-items: center;
-  margin: $inner-padding 0;
+$label-width: 60px;
+$long-label-width: $label-width + 10px;
 
-  & > .indicator {
-    width: $can-drop-size;
-    height: $can-drop-size;
-    border-radius: 10px;
-  }
-
-  &.can-drop > .indicator {
+#inspector {
```

Comment:
A css best practice is to never use IDs and never use html elements (h2, h3, dl, dt, button) and use classes instead. That is to prevent cases where you might the look of an element on a different element.  
So I might want the look of an h2, on my span.